plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -89,6 +98,15 @@ protected Response handle(final Request request) throws IOException {
                     return newError(request.getId(), RestStatus.FORBIDDEN, "AccessDenied", "Bad session token", "");
                 }
                 permittedBucket = temporaryBucketName;
+            } else if (authorization != null && authorization.contains("securitycredentials42_KEYID")) {
```

Comment:
Could we extract `securitycredentials42` out as a named constant so it's clear why it is what it is?